### PR TITLE
Improve contributing experience

### PR DESCRIPTION
**Pull request by sigmaSd**
This pr applies a bunch of simple quality of life patches.

I believe if you accept this pr it should easier a lot  the contributing experience.

### CODE REVIEW

File: src/application.rs [hunk 1]
```diff
@@ -69,66 +75,86 @@ impl Application {
...
+        self.network
+            .listen_udp_multicast(self.discovery_addr)
+            .unwrap();
 
         let discovery_endpoint = self.network.connect_udp(self.discovery_addr).unwrap();
-        self.network.send(discovery_endpoint, NetMessage::HelloLan(self.user_name.clone(), server_port)).unwrap();
+        self.network
+            .send(
+                discovery_endpoint,
+                NetMessage::HelloLan(self.user_name.clone(), server_port),
```
Comment (lemunozm) [inline src/application.rs]: I suppose that this style has been created by rust fmt. For example, in cases like this, I think that is much more readable to extract the message into a variable `let message = NetMesssge::HelloLan(...)` and let the send() method in one line. What do you think?

File: README.md [hunk 2]
```diff
@@ -33,5 +33,14 @@ $ termchat
 
 to open the application in your terminal.
 
-By default, your computer user name is used.
-You can rename your this name using the argument `-u <name>`, or modified the multicast discovery address (see the application help `--help`).
+By default, your computer user name is used. You can use a diffrent username with `-u <name>`
```
Comment (lemunozm) [inline README.md]: Minnor typo: diffrent to different

File: README.md [hunk 3]
```diff
@@ -33,5 +33,14 @@ $ termchat
...
+
+Also you can modify the multicast discovery address with `-d <address>` 
+
+(see the application help for more info `--help`).
+
+## Frequently Asked Questions
```
Comment (lemunozm) [inline README.md:42]: Cool!

File: src/application.rs (hunk 1 above)
Comment (sigmaSd) [reply src/application.rs]: Yes I agree

Personally I'm ok with any  style as long as it is automatically checked. 

If you have other suggestions I'll also apply them.

File: src/application.rs [hunk 4]
```diff
@@ -69,66 +75,79 @@ impl Application {
...
                                 state.input_write(character);
                             }
-                        },
+                        }
                         KeyCode::Enter => {
                             if let Some(input) = state.reset_input() {
-                                let message = LogMessage::new(format!("{} (me)", self.user_name), MessageType::Content(input.clone()));
-                                self.network.send_all(state.all_user_endpoints(), NetMessage::UserMessage(input)).unwrap();
+                                let message = LogMessage::new(
+                                    format!("{} (me)", self.user_name),
```
Comment (lemunozm) [inline src/application.rs:133]: Total personal comment: I think that putting the `else` after the `}` in the same line breaks the indent parallelism between if and else. If I not remember bad, in the fmt file there is an option to change it. I want to have time to create this file. I note it :)

File: .github/workflows/rust.yml [hunk 5]
```diff
@@ -0,0 +1,61 @@
+#Copy paste from crossterm
+name: termchat ci
```
Comment (lemunozm) [inline .github/workflows/rust.yml:2]: Good idea!

**Review by lemunozm — CHANGES_REQUESTED** [review]
I agree the PR after some minor suggested changes. Anyway, when I have time to create a custom fmt file and analyze it, I will probably modify some style things.

Thanks for your collaboration!

File: src/application.rs (hunk 4 above)
Comment (lemunozm) [reply src/application.rs:133]: I saw that this comment is bad placed :S

It is related to `} else {` lines. Anyway, if you want, do not care about it. I will modify it when I establish a rust fmt file.

Thanks!

Comment (sigmaSd) [reply src/application.rs:133]: This is what want probably https://rust-lang.github.io/rustfmt/?version=master&search=#control_brace_style

**Pull request merged**